src/exercise/06.md

```diff
@@ -90,8 +90,8 @@ make this work:
    easily via `const isValid = value === value.toLowerCase()`
 3. If there's an error, set the error state to `'Username must be lower case'`.
    (💰 here's how you do that:
-   `setError(isValid ? null : 'Username must be lower case')`)
-4. Finally, display the error in an element.
+   `setError(isValid ? null : 'Username must be lower case')`) and disable the submit button.
+4. Finally, display the error in an element with [role="alert"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Alert_Role).
```

Comment:
Awesome, thank you! I'm also going to update the test to give a more useful warning. Decided to reword this a bit:

```suggestion
   `setError(isValid ? null : 'Username must be lower case')`) and disable the submit button.
4. Finally, display the error in an element

You may consider adding a `role="alert"` to the element you use to display the error to assist with screen reader users.
```